app/index.js

```diff
@@ -99,105 +86,8 @@ const prefsRestartLastValue = {}
-
-  const appState = AppStore.getState().toJS()
-  appState.perWindowState = perWindowState
-
-  const receivedAllWindows = perWindowState.length === BrowserWindow.getAllWindows().length
-  if (receivedAllWindows) {
-    clearTimeout(saveAppStateTimeout)
-  }
-
-  if (!forceSave && !receivedAllWindows) {
-    return
-  }
-
-  return SessionStore.saveAppState(appState, shuttingDown).catch((e) => {
-    logSaveAppStateError(e)
-  }).then(() => {
-    if (receivedAllWindows || forceSave) {
-      sessionStateStoreComplete = true
-    }
-
-    if (sessionStateStoreComplete) {
-      if (shuttingDown) {
-        // If the status is still UPDATE_AVAILABLE then the user wants to quit
-        // and not restart
-        if (appState.updates && (appState.updates.status === UpdateStatus.UPDATE_AVAILABLE ||
-            appState.updates.status === UpdateStatus.UPDATE_AVAILABLE_DEFERRED)) {
-          // In this case on win32, the process doesn't try to auto restart, so avoid the user
-          // having to open the app twice.  Maybe squirrel detects the app is already shutting down.
-          if (process.platform === 'win32') {
-            appState.updates.status = UpdateStatus.UPDATE_APPLYING_RESTART
-          } else {
-            appState.updates.status = UpdateStatus.UPDATE_APPLYING_NO_RESTART
-          }
-        }
-
-        // If there's an update to apply, then do it here.
-        // Otherwise just quit.
-        if (appState.updates && (appState.updates.status === UpdateStatus.UPDATE_APPLYING_NO_RESTART ||
-            appState.updates.status === UpdateStatus.UPDATE_APPLYING_RESTART)) {
-          Updater.quitAndInstall()
-        } else {
-          app.quit()
-        }
-      } else {
-        const cb = sessionStateStoreCompleteCallback
-        sessionStateStoreCompleteCallback = null
-        cb()
-      }
-    }
-  })
-}
-
-/**
- * Saves the session storage for all windows
- */
-const initiateSessionStateSave = () => {
-  sessionStoreQueue.push((cb) => {
-    sessionStateStoreComplete = false
-    sessionStateStoreCompleteCallback = cb
-
-    perWindowState.length = 0
-    // quit triggered by window-all-closed should save last window state
-    if (lastWindowClosed && lastWindowState) {
-      perWindowState.push(lastWindowState)
-      saveAppState(true)
-    } else if (BrowserWindow.getAllWindows().length > 0) {
-      ++requestId
-      BrowserWindow.getAllWindows().forEach((win) => win.webContents.send(messages.REQUEST_WINDOW_STATE, requestId))
-      // Just in case a window is not responsive, we don't want to wait forever.
-      // In this case just save session store for the windows that we have already.
-      saveAppStateTimeout = setTimeout(() => {
-        saveAppState(true)
-      }, appConfig.quitTimeout)
-    } else {
-      saveAppState()
-    }
-  })
-}
-
 // exit cleanly on signals
-['SIGTERM', 'SIGHUP', 'SIGINT', 'SIGBREAK'].forEach((signal) => {
+;['SIGTERM', 'SIGHUP', 'SIGINT', 'SIGBREAK'].forEach((signal) => {
```

Comment:
Yeah, I think it is. Because we don't use `;` and you have `}` in the last line and this line starts with `[`, so this could lead into errors